internal/pkg/cli/init_test.go

```diff
@@ -12,6 +12,20 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+type mockAppDeployer struct{}
+
+func (mad mockAppDeployer) init() error {
```

Comment:
lol `mad`

Reply:
tell us how you really feel @sonofachamp

Reply:
It may or may not have been related to fighting `gomock`